libs/ui-lib/lib/cim/components/InfraEnv/InfraEnvAgentTable.tsx

```diff
@@ -162,6 +163,7 @@ const InfraEnvAgentTable: React.FC<InfraEnvAgentTableProps> = ({
       hostActions.canEditHostname,
       hostActions.canEditBMH,
       agentStatuses,
+      bmhStatuses,
```

Comment:
We should memoize the `const bmhStatuses = bmhStatus(t);` call above, as it is now used as a dependency for this `useMemo`